src/couch_mrview_http.erl

```diff
@@ -229,7 +229,8 @@ do_all_docs_req(Req, Db, Keys, NS) ->
     end,
     Args = Args1#mrargs{preflight_fun=ETagFun},
     {ok, Resp} = couch_httpd:etag_maybe(Req, fun() ->
-        VAcc0 = #vacc{db=Db, req=Req},
+        Max = config:get_integer("httpd", "chunked_response_buffer", 1490),
```

Comment:
Heads-up for Cloudant folks, please don’t refer to private repos / issues here, thanks :)